block_control/block_control.F90

```diff
@@ -198,10 +198,9 @@ subroutine define_blocks_packed (component, Block, isc, iec, jsc, jec, &
       nblks = 1
       blksz = tot_pts
     else
-      if (mod(tot_pts,blksz) .eq. 0) then
-        nblks = tot_pts/blksz
-      else
-        nblks = ceiling(real(tot_pts)/real(blksz))
+      nblks = tot_pts/blksz
```

Comment:
@bensonr, just wanted to make sure you're ok with this change

Reply:
@mlee03 - it looks good to me.